src/extension.ts

```diff
@@ -96,18 +166,37 @@ class PyPICodeLens extends vscode.CodeLens {
 
 class PyPICodeLensProvider implements vscode.CodeLensProvider<PyPICodeLens> {
     provideCodeLenses(document: vscode.TextDocument): PyPICodeLens[] {
+        const codeLensEnabled = vscode.workspace.getConfiguration('pypiAssistant').get('codeLens')
         const codeLenses: PyPICodeLens[] = []
-        if (vscode.workspace.getConfiguration('pypiAssistant').get('codeLens')) {
+
+        if (!codeLensEnabled) {
+            return codeLenses
+        }
+
+        if (document.languageId === 'toml' && document.fileName.toLowerCase().endsWith('pyproject.toml')) {
```

Comment:
Because this condition is used twice and is pretty long, it'd be good for readability to refactor it into an `isPyProjectToml` function